shipstation_integration/customer.py

```diff
@@ -19,8 +19,9 @@ def update_amazon_order(
     existing_so_doc: "SalesOrder" = frappe.get_doc("Sales Order", existing_so)
 
     email_id, user_name = parse_addr(existing_so_doc.amazon_customer)
-    if email_id:
-        contact = create_contact(order, email_id)
+    phone_no = order.ship_to.phone
```

Comment:
I think you'll have to check if the `ship_to` object exists first, similar to line 55